jetty-ee9/jetty-ee9-webapp/src/main/java/org/eclipse/jetty/ee9/webapp/MetaInfConfiguration.java

```diff
@@ -180,7 +182,8 @@ public void findAndFilterContainerPaths(final WebAppContext context) throws Exce
         {
             List<Path> matchingBasePaths =
                 Stream.of(modulePath.split(File.pathSeparator))
-                    .map(URIUtil::toURI)
+                    .map(resourceFactory::newResource)
```

Comment:
We should modify the method's javadoc to specify it mounts jars.

Reply:
done in ee10